lib/galaxy/authnz/managers.py

```diff
@@ -260,6 +262,23 @@ def callback(self, provider, state_token, authz_code, trans, login_redirect_url)
             log.exception(msg)
             return False, msg, (None, None)
 
+    def logout(self, provider, trans, post_logout_redirect_url=None):
+        try:
+            # check if logout is enabled for this idp and return false if not
+            unified_provider_name = self._unify_provider_name(provider)
+            if self.oidc_backends_config[unified_provider_name]['enable_idp_logout'] is False:
+                return False, "IDP logout is not enabled for {}".format(provider), None
```

Comment:
Are the return values for this documented somewhere?

Reply:
Good catch, I'll document.